app/controllers/root_controller.rb

```diff
@@ -354,8 +354,7 @@ def custom_format_locals
   end
 
   def setup_navigation_helpers
-    content_item = content_store.content_item("/" + params[:slug])
-    @navigation_helpers = GovukNavigationHelpers::NavigationHelper.new(content_item)
+    super("/" + params[:slug])
```

Comment:
Can you rename the `setup_navigation_helpers` here and call `setup_navigation_helpers` from that? That way we don't have to use `super`.